### ISSUE TITLE
chromum vs. chromium-browser

### ISSUE THREAD

**Issue by haraldschilly** [sagemathinc/cocalc#7801]
This is something I noticed when working on Ubuntu 24.04 based software environment for onprem – but we'll probably hit it in production soon as well: There is not only "chromium-browser" but also just "chromium". The reason is installing it as a deb package, and not via snap. I followed this https://askubuntu.com/questions/1204571/how-to-install-chromium-without-snap/1511695#1511695 

So, I think we should support `chromium` as well, in all places where there is "chromium-browser" in the source code. Which means

* [ ] expand the check for PDF conversion (project capabilities)
* [ ] use that for actually doing the call to convert html to PDF
* [ ] and for x11: add a "chromium" app

**Comment by williamstein** [comment]
For Ubuntu 24.04 I just install chromium-browser, which works fine on both x86_64 and arm64:

https://github.com/sagemathinc/cocalc-docker-secret/blob/db3449a113c0c995eab0a4541612791e5dffbfbe/Dockerfile#L252

This follows the directions you also found.  It works fine, I think and provides chromium-browser without snap, which is needed inside docker.

```sh
RUN \
    add-apt-repository -y ppa:saiarcot895/chromium-beta \
 && apt-get update \
 && DEBIAN_FRONTEND=noninteractive apt install -y chromium-browser
```

I do not in any way understand the point of this issue.  Why also support chromium?  Makes no sense to me.

Basically, you link to some directions for installing chromium-browser which work very well to install chromium-browser.  You then say * There is not only "chromium-browser" but also just "chromium". * but I have no idea why you say that or what it has to do with the linked directions.

**Comment by williamstein** [comment]
I looked at https://askubuntu.com/questions/1204571/how-to-install-chromium-without-snap/1511695#1511695 again.  I guess they think you have to instead install a package called "chromium" now, **instead of** chromium-browser.   In fact if you install the chromium package, you get this:
```sh
~/cocalc/src$ ls -lht /usr/bin/chromium*
lrwxrwxrwx 1 root root    8 Aug 25 00:34 /usr/bin/chromium-browser -> chromium
-rwxr-xr-x 1 root root 5.0K Aug 23 09:02 /usr/bin/chromium
```

So no need for us to worry.
